tests/utilities/test_storage.py

```diff
@@ -61,6 +61,20 @@ def flow_path(self, tmpdir):
 
         return full_path
 
+    @pytest.fixture
+    def flow_path_with_additional_file(self, tmpdir):
+        contents = """from prefect import Flow\nfrom pathlib import Path\nwith open(str(Path(__file__).resolve().parent)+"/test.txt", "r") as f:\n\tname=f.read()\nf2=Flow(name)"""
```

Comment:
Can you change this to a multiliner with `textwrap.dedent?` so it's easy to read?